### PR TITLE
Updating Jakarta Server Faces 3.0 Release Notes

### PR DESCRIPTION
**Pull request by gabrielbussolo**
complementing pull request #454 and contributing to #428

~ps: I'm leaving it as a draft so the task progress can be tracked and errors and suggestions can be reviewed earlier~

### CODE REVIEW

File: install/jsf/docs/README.txt [hunk 1]
```diff
@@ -13,8 +13,8 @@
...
 # SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 #
-The Java Server Faces Technology Compatibility Kit 2.3 User's Guide provides
-the information you need to install, configure, and run the JSF
+The Java Server Faces Technology Compatibility Kit 3.0 User's Guide provides
+the information you need to install, configure, and run the Server Faces
 TCK. The User's Guide also contains the rules you need to pass for
 certification. The guide is available in PDF format at
-guides/ug/JSF-TCK_Users_Guide.pdf relative to this README file.
+guides/ug/Jakarta-Server-Faces-TCK-Users-Guide.pdf relative to this README file.
```
Comment (alwin-joseph) [inline install/jsf/docs/README.txt:20]: relative path would be pdf-usersguide/

File: install/jsf/docs/README.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```
Comment (edbratt) [inline install/jsf/docs/README.txt:2]: Please change date string to ... (c) 2018, 2020 Oracle and/or ....

File: install/jsf/docs/index.html [hunk 3]
```diff
@@ -1,70 +1,113 @@
...
 
 <h2>JT Harness Documentation</h2>
-<p>The online version of the JT Harness version 5.0 documentation is available <a href="https://wiki.openjdk.java.net/display/CodeTools/Documentation">here</a>.</p>
+<p>The online version of the JT Harness version 5.0 documentation is available <a
+        href="https://wiki.openjdk.java.net/display/CodeTools/Documentation">here</a>.</p>
 
 <hr>
-<p><small><address>Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.</address></small></p>
+<p><small>
+    <address>Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.</address>
```
Comment (edbratt) [inline install/jsf/docs/index.html:110]: Please update second date from 2018 to 2020

File: install/jsf/docs/README.txt (hunk 2 above)
Comment (gabrielbussolo) [reply install/jsf/docs/README.txt]: thanks!

File: install/jsf/docs/index.html (hunk 3 above)
Comment (gabrielbussolo) [reply install/jsf/docs/index.html]: thanks !

**Pull request merged**